README.rst

```diff
@@ -541,6 +554,11 @@ Choose from the list of available fixers:
                         the class name should match
                         the file name. (Risky fixer!)
 
+* **random_api_migration**
```

Comment:
I have been sure you are sending the PR to 1.12 branch, as stated in contributing file...
And on 1.12 it would be static